tests/components/hue/test_config_flow.py

```diff
@@ -199,13 +279,16 @@ async def test_flow_link_timeout(hass):
             const.DOMAIN, context={"source": "user"}
         )
 
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input={"id": mock_bridge.id}
+    )
+
     result = await hass.config_entries.flow.async_configure(
         result["flow_id"], user_input={}
     )
 
-    assert result["type"] == "form"
-    assert result["step_id"] == "link"
-    assert result["errors"] == {"base": "linking"}
+    assert result["type"] == "abort"
+    assert result["reason"] == "cannot_connect"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5795327)